stacks/nodejs/3.0.0/devfile.yaml

```
@@ -0,0 +1,67 @@
+schemaVersion: 3.0.0
+metadata:
+  name: nodejs
+  displayName: Node.js Runtime
+  description: Node.js 18 application
+  icon: https://nodejs.org/static/images/logos/nodejs-new-pantone-black.svg
+  tags:
+    - Node.js
+    - Express
+    - ubi8
+  projectType: Node.js
+  language: JavaScript
+  version: 2.2.0
```

Need to swap the `schemaVersion` with the stack `version`.

```suggestion
schemaVersion: 2.2.0
metadata:
  name: nodejs
  displayName: Node.js Runtime
  description: Node.js 18 application
  icon: https://nodejs.org/static/images/logos/nodejs-new-pantone-black.svg
  tags:
    - Node.js
    - Express
    - ubi8
  projectType: Node.js
  language: JavaScript
  version: 3.0.0
```